app/views/events/_event.html.erb

```diff
@@ -119,4 +141,42 @@
   </div>
 <% end %>
 
+<p>Here is the crucial info</p>
+<% if @event.theme_image_url %>
+  <%= image_tag @event.theme_image_url, style: 'width: 100%; height: 200px' %>
+<% end %>
+
+<div class="row mt-3">
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-12">
+    <p><b><%= @event.name %>: <%= @event.theme %></b></p>
+    <%= simple_format(@event.theme_details) %>
+  </div>
+
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-12">
+    <p>Where: <%= @event.location %></p>
+    <p>Map: <%= link_to 'Google Maps Location', @event.maps_location_url %></p>
+    <% if @event.eventbrite_event.eventbrite_event.start.local.nil? %>
+      <p>When: <%= simple_format(@event.event_timings) %></p>
+    <% else %>
+      <p>When: <%= Date.parse(@event.eventbrite_event.eventbrite_event.start.local).strftime("%A %d %B %Y") %> <%= simple_format(@event.event_timings) %></p>
+    <% end %>
+    <p>Further Information: <%= @event.further_information %></p>
+    <p>
+      Standard ticket: £25 (+£2.80 booking fee)
```

Comment:
why is this hard coded when we have so much else in the event model?

Reply:
I am not sure, will update